### PR TITLE
XIVY-9644 Load add-opens etc from engine jvm-module.options file

### PR DESCRIPTION
**Pull request by ivy-lli**

### CODE REVIEW

File: src/test/java/ch/ivyteam/ivy/maven/test/TestSetupIvyTestPropertiesMojo.java [hunk 1]
```diff
@@ -66,7 +66,7 @@ public void engineModuleHintsSharedAsProperty() throws Exception {
 
     mojo.execute();
 
-    assertThat(getProperty(Property.MAVEN_TEST_ARGLINE)).contains(" --add-opens ");
+    assertThat(getProperty(Property.MAVEN_TEST_ARGLINE)).contains(" --add-opens=");
```
Comment (ivy-lli) [inline src/test/java/ch/ivyteam/ivy/maven/test/TestSetupIvyTestPropertiesMojo.java:69]: before the --add-opens / --add-export properties where added as `--add-opens blablabla` and now `--add-opens=blablabla` is there an difference?

Comment (ivy-rew) [reply src/test/java/ch/ivyteam/ivy/maven/test/TestSetupIvyTestPropertiesMojo.java:69]: I think the JVM args for surefire work exactly like any other argument for a normal 'java' binary. So if `--add-opens xyz` and `--add-opns=XYZ` are both interpreted by the JVM I'm fine with the change

**Review by ivy-rew — APPROVED** [review]

**Comment by ivy-rew** [comment]
I'd not expect a big impact by this change: still please consider to verify locally, whether you can still build 'demo-projects' and 'project-build-examples' with this new version of the plugin ... before rolling it out to the public.

**Comment by ivy-lli** [comment]
> I'd not expect a big impact by this change: still please consider to verify locally, whether you can still build 'demo-projects' and 'project-build-examples' with this new version of the plugin ... before rolling it out to the public.

demo-projects and the project-build-examples seems to build/test without problems.

**Pull request merged**